include/votca/csg/basebead.h

```diff
@@ -110,40 +128,46 @@ class BaseBead : public TopologyItem,
    * \return reference to position
    */
   virtual vec &Pos() {
-    assert(_bPos);
-    return _pos;
+    assert(bead_position_set_);
+    return bead_position_;
   }
 
   /** does this configuration store positions? */
-  bool HasPos() const { return _bPos; }
+  bool HasPos() const { return bead_position_set_; }
 
   /** set has position to true */
-  void HasPos(bool true_or_false) { _bPos = true_or_false; }
+  void HasPos(bool true_or_false) { bead_position_set_ = true_or_false; }
 
-protected:
+ protected:
   BaseBead()
-      : TopologyItem(nullptr), MoleculeItem(nullptr), 
-      _mass(0.0), _bPos(false){};
+      : topology_item_(nullptr),
+        molecule_item_(nullptr),
+        mass_(0.0),
+        bead_position_set_(false){};
 
+  TopologyItem topology_item_;
+  MoleculeItem molecule_item_;
 
-  std::weak_ptr<BeadType> _type;
+  Identity<int> id_;
+  Name name_;
+  std::weak_ptr<BeadType> type_;
 
-  double _mass;
-  vec _pos;
+  double mass_;
+  vec bead_position_;
 
-  bool _bPos;
+  bool bead_position_set_;
 };
 
-inline void BaseBead::setPos(const vec &r) {
-  _bPos = true;
-  _pos = r;
+inline void BaseBead::setPos(const vec &bead_position) {
+  bead_position_set_ = true;
+  bead_position_ = bead_position;
 }
 
 inline const vec &BaseBead::getPos() const {
-  assert(_bPos);
-  return _pos;
-}
-}
+  assert(bead_position_set_);
```

Comment:
make it `assert(bead_position_set_ & "Position not set")` an assert without a message is really hard to debug